test/command-palette-view.test.js

```diff
@@ -39,39 +74,29 @@ describe('CommandPaletteView', () => {
 
     describe('when no element has focus', () => {
       it('uses the root view as the element to display and dispatch events to', async () => {
+        const findCommandsSpy = sandbox.spy(atom.commands, 'findCommands')
+        const findKeyBindingsSpy = sandbox.spy(atom.keymaps, 'findKeyBindings')
+
         document.activeElement.blur()
         const commandPalette = new CommandPaletteView()
         await commandPalette.toggle()
-        const keyBindings = atom.keymaps.findKeyBindings({target: workspaceElement})
```

Comment:
I cleaned these up to directly express what's in the description.

Let me know if you want me to revert this or include as a separate PR.